packages/firestore/test/unit/remote/bloom_filter_golden_test_data/index.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export { default as testDataCount1Rate1 } from './Validation_BloomFilterTest_MD5_1_1_bloom_filter_proto.json';
```

Comment:
Is is possible to do

```ts
export { default as testDataCount1Rate1 as TestDataType } from '...';
```

(i.e. add `as TestDataType`) so that it can be exported with the right type? That would save doing the cast at all of the call sites.

Reply:
"as" inside the export {} doesn't do type casting. The interfaces TestDataType and TestResultType were added to accent the data types we are dealing in the test, but it is not required. Typescript will automatically add type to json files imported:
![Screen Shot 2022-11-27 at 11 44 20 AM](https://user-images.githubusercontent.com/107142260/204156316-7082736a-e26b-488b-baf0-68a514ad7d55.png)
I can remove the type casting if it is noisy.

Reply:
Thank you for removing the type casting. Much better.